Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -35,6 +35,8 @@ public enum SentrySDK {
         start(options)
     }
 
+    static let fatalErrorMessageHandle = getFatalErrorMessageHandle()
```

Comment:
This is evaluated on first use IIRC, so this is unclear when it would be evaluated.

Reply:
Is there a better way?

Reply:
Actually, line 78 below accesses this when the on crash handler is set by `sentry_options_set_on_crash` which is when sentry is initialized. So, it seems okay to me.